core/frontend/src/ViewState.ts

```diff
@@ -256,7 +256,7 @@ export abstract class ViewState extends ElementState {
       await subcategories.promise;
   }
 
-  /** Returns true if all [[TileTree]]s required by this view have been loaded. */
+  /** Returns true if all [[TileTree]]s required by this view have been loaded.  Note that the map tile trees associated to the viewport rather than the view. */

```

Comment:
Can you add a link to `Viewport.areAllTileTreesLoaded` here and a note to NextVersion.md describing the change in behavior?

Reply:
Yes - I wasn't sure of this one.  Fixed.